app/controllers/csvs_controller.rb

```diff
@@ -0,0 +1,70 @@
+class CsvsController < ApplicationController
+
+  def index
+    respond_to do |format|
+      @csv_exports = CsvExport.where(user_id: current_user.id)
+      format.html
+      format.json { render json: @csv_exports }
+    end
+  end
+  
+  def show
+    respond_to do |format|
+      @csv_export = CsvExport.find_by_id(params[:id])
+      format.html { not_found if @csv_export.nil? }
+      format.json do
+        if @csv_export.nil?
+          not_found_json
+        else
+          render json: @csv_export
+        end
+      end
+      format.csv { download(params) }
+    end
+  end
+
+  def create
+    csv_export = CsvExport.create(user: current_user, content: params[:content])
```

Comment:
since it is not dealing with errors a `create!` would show us the errors if something goes wrong